AP_Randomizer/apworld/rules_normal.py

```diff
@@ -184,6 +238,66 @@ def __init__(self, world) -> None:
         }
 
         location_clauses = {
+            "Empty Bailey - Solar Wind": lambda state:
+                self.has_slide(state),  # to consider: damage boosting w/ crouch
+            "Empty Bailey - Cheese Bell": lambda state:  # TODO consider to/from center steeple
+                self.can_slidejump(state)  # untested
+                and (
+                    self.kick_or_plunge(state, 2)
+                    or self.has_gem(state))
+                or self.get_kicks(state, 3),
```

Comment:
logic here seems a little too permissive for normal difficulty. eg how are you making it there with just 3 kicks?

Reply:
you can backflip + pretty much spam the three kicks and get up just fine
also i believe slightly more thought into your kick timing gets you up without a backflip

backflips are in normal logic but if you want to argue that them + other movement should be in hard idc

Reply:
gem + kickorplunge2 without cling climbing does not seem reasonable imo
but are you suggesting remove the solar ultra route here?

Reply:
its really not terrible but arguably not normal I guess. is the intention a solar ultra? is that in normal logic?

Reply:
yes,
and i assumed so, potentially only going backwards? which would make this not in logic so we could remove it

Reply:
looking through all the logic, I assumed solar ultras basically weren't in logic at all. they generally aren't expected in normal/hard, and expert can do slide ultras

Reply:
```suggestion
            "Empty Bailey - Cheese Bell": lambda state:  # TODO consider to/from center steeple
                self.get_kicks(state, 3),
```
feels weird to only have one way into cheese bell though, especially because it seems primed to have solar ultras be the vanilla way to check the location, i'll give it some thought but probably make this change

Reply:
I always though the vanilla way involved gem lol https://youtu.be/Ur7LppMNA4c

Reply:
idk why but i'm not getting that kick+plunge to work, 2 kicks seems to be fine

Reply:
wait I just realized for gem + 1 kick + sunsetter you could start with sunsetter to get more height which would probably make it easier

Reply:
just gonna commit this and then we can revisit the solar ultras with everything else
```suggestion
            "Empty Bailey - Cheese Bell": lambda state:  # TODO consider to/from center steeple
                self.can_slidejump(state)  # untested
                and (
                    self.kick_or_plunge(state, 2)
                    or self.has_gem(state))
                or self.get_kicks(state, 3)
                or self.has_gem(state) and self.kick_or_plunge(state, 2),
```

Reply:
backwards ultra 2 kicks is possible but kinda difficult, your kicks have to be pretty good